nautobot/users/factory.py

```diff
@@ -3,19 +3,27 @@
 from django.contrib.auth import get_user_model
 import factory
 
-from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator
+from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator, random_instance, UniqueFaker
+from nautobot.users.models import SavedView
 
 User = get_user_model()
 
+VIEW_NAMES = [
+    "circuits:circuit_list",
+    "dcim:device_list",
+    "dcim:location_list",
+    "ipam:ipaddress_list",
+]
```

Comment:
Could we maybe instead run a FeatureQuery for all SavedView-capable models and just pick one of them at random and get its list view?